com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -266,59 +385,61 @@ static Vector3Int ProbeCountToDataLocSize(int numProbes)
             return new Vector3Int(width, height, depth);
         }
 
-        public static DataLocation CreateDataLocation(int numProbes, bool compressed, ProbeVolumeSHBands bands, string name, out int allocatedBytes)
+        public static Texture CreateDataTexture(int width, int height, int depth, GraphicsFormat format, string name, bool allocateRendertexture, ref int allocatedBytes)
```

Comment:
Since we need some pools to have read write enabled and others to be uploadable from cpu, some are rendertexture and some are texture3D.
For example, the pool texture used at runtime is now a rendertexture because we need to write to it in the blending compute shader